Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx

```diff
@@ -98,16 +98,16 @@ const ContestDetailsPage = () => {
                 <Link
                   className={`${isCompete
                       ? styles.greenColor
-                      : ''}`}
+                      : styles.blueColor}`}
                   to={getContestsResultsUrl(id!, isCompete
                       ? 'compete'
                       : 'practice', true)}
                 >
                     <i className="fas fa-user" />
                     <div className={`${styles.underlinedBtnText}`}>
                         { isCompete
-                            ? 'compete'
-                            : 'practice'}
+                            ? 'Compete'
+                            : 'Practice'}
```

Comment:
Do we have extracted constants for these. I think they are used in many places

Reply:
I moved them to a seperate file, since the dublication was happening because of the usage of those words in the buttons, which are not ContestButton.tsx, there we have constants for those specific strings, but are not exported, since logically they are related to those buttons and it wouldnt make sense to export them as a global constant, Im not sure if it would be the best approach to have a global string for this specific case only